docs/webhooks/index.html

```diff
@@ -328,7 +449,127 @@ <h3 id="companyTestSessionFinished">companyTestSessionFinished</h3>
   }
 };</code></pre>
                 </div>
-		<h2>Events for CodeSignal Certify</h2>    
+<div class="subsection">
+   <h3 id="auditorworkflow">CodeSignal Auditor workflow</h3><br>
+       <img alt="Diagram of webhooks triggered on CodeSignal auditor flow" width="100%" src="../images/auditor-events.png" />
+</div>
+
+                <div class="subsection">
+                    <h4 id="preScreenResultNotVerified">preScreenResultNotVerified</h4>
+                    <p>
+                        Fired when an auditor rejects a proctored test session.
```

Comment:
```suggestion
                        Fired when a CodeSignal auditor does not verify a proctored test session.
```

Not sure about this one, sounds a bit funny 🤔

Reply:
How can we say reject without reject? Declines?

Reply:
"Marks session as not verified"? Englishsky is hard 😅  Perhaps @menewman has an idea?

Reply:
Yeah, it's tricky because "does not verify" can sound passive (like they took _no_ action or didn't review it at all), and we want to convey that they took the _proactive_ step of flagging a session for suspicious behavior.

I agree it helps to rephrase this just as you suggested, with "marks ... as not verified" conveying that it's an _active_ thing, not passive.

e.g., "Fired when a session is marked as not verified by CodeSignal."

If we want to elaborate a little, we could append:
"This means that either the session has been flagged for suspicious behavior, or a technical issue has been identified during the session (in which case the candidate will be automatically granted a retake)."

Note that I took out the word "auditor" from my version. That's intentional, because it might be nice to obscure the actual verification mechanism in cases where a human may or may not have actually reviewed the session.

Reply:
I like that.